### DISCUSSION TITLE
Authorization Service

### DISCUSSION THREAD

**Discussion by phil-hotchkiss** [mia-platform/community#396] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

12.2.0

### Description

Hi there, 

I am a little confused about the authorization service. Within my project, I have defined an endpoint that has the `Authorization Required` checkbox **unchecked**, but it appears that the Authorization Service is getting called for every request. I am receiving a 401 because I am not supplying an access token it seems. I was expecting that in defining an endpoint where the `Authorization Required` checkbox is unchecked that an access token would not need to be supplied to the endpoint. Is this a false assumption? I've wired up the authorization service to work with the JWT Token Validator. 

Thank you!

### Actual Outcome

401 Unauthorized when hitting an endpoint with 'Authorization Required' checkbox unchecked. 

### Expected Outcome

I expected this request to be routed to the appropriate microservice when the `Authorization Required` checkbox is not checked, regardless of an authorization token being provided.

**Comment by davidebianchi** [comment]
Hi @phil-hotchkiss! Yes, the authorization service is called for every request, since it is the server which evaluate the permissions needed to forward the API call (including whether or not authentication is required).
It is correct, if the `Authorization Required` checkbox is unchecked is not necessary to pass an access token.
There are other settings, for example `API Key required` or `User Group Permission` one? What is the response of the authorization-service?

**Comment by phil-hotchkiss** [comment]
I was able to get this working as expected. Sorry for the confusion.
